enterprise/backend/test/metabase_enterprise/serialization/v2/extract_test.clj

```diff
@@ -545,14 +547,15 @@
                                                              :human_readable_field_id cust-name}]
       (testing "dimensions without foreign keys are inlined into their Fields"
         (let [ser (serdes/extract-one "Field" {} (t2/select-one Field :id email-id))]
-          (is (schema= {:serdes/meta   (s/eq [{:model "Database" :id "My Database"}
-                                              {:model "Table"    :id "Schemaless Table"}
-                                              {:model "Field"    :id "email"}])
-                        :dimensions    [{(s/optional-key :human_readable_field_id) [(s/maybe s/Str)]
-                                         :created_at                               LocalDateTime
-                                         s/Keyword                                 s/Any}]
-                        s/Keyword      s/Any}
-                       ser))
+          (is (malli= [:map
+                       [:serdes/meta [:= [{:model "Database", :id "My Database"}
+                                          {:model "Table", :id "Schemaless Table"}
+                                          {:model "Field", :id "email"}]]]
+                       [:dimensions  [:sequential
+                                      [:map
+                                       [:created_at (ms/InstanceOfClass LocalDateTime)]
+                                       [:human_readable_field_id {:optional true} [:maybe [:sequential :string]]]]]]]
```

Comment:
It actually needs to be `[:maybe [:sequential [:maybe :string]]]`, because Schema sequences like `[x]` accept `nil`, while Malli ones do not. `[:maybe [:sequential x]]` is the Malli equivalent of Schema `[x]`

Reply:
fixed